core/src/main/java/edu/wpi/grip/core/operations/network/ros/ROSManager.java

```diff
@@ -45,12 +45,12 @@
    */
   private static class GripPublisherNode extends AbstractNodeMain {
     private static final GraphName GRIP_ROOT = GraphName.of("GRIP/publisher");
-    private final java.lang.String publishType;
+    private final String publishType;
```

Comment:
I feel like I did this intentionally. There was some stupid naming collision.

Reply:
I suspected that, but there doesn't seem to have been any problems caused by removing the qualifier.

OT: Can the grip_msg library and friends use Java modules?

Reply:
> Can the grip_msg library and friends use Java modules?